.github/workflows/main.yml

```diff
@@ -31,7 +31,12 @@ jobs:
       - name: Spellcheck
         run: npm run check:spelling
       - name: Link checker
-        run: npm run check:links -- --current-apis --qiskit-release-notes
+        run: |
+          npm run check:links -- \
+            --current-apis \
+            --qiskit-release-notes \
+            --historical-apis \
+            --skip-broken-historical
```

Comment:
This will make CI for every PR a little slower. But given that we're making changes to the API generation script the next two months, it's probably worth it. We could always remove it if CI is getting too slow.